packages/twenty-server/src/engine/core-modules/auth/services/social-sso.service.ts

```diff
@@ -55,14 +55,17 @@ export class SocialSsoService {
             },
           },
         },
-        relations: ['workspaceUsers', 'workspaceUsers.user'],
+        relations: ['workspaceUsers', 'workspaceUsers.user', 'trustDomains'],
       });
 
       return workspace ?? undefined;
     }
 
-    return await this.workspaceRepository.findOneBy({
-      id: workspaceId,
+    return await this.workspaceRepository.findOne({
+      where: {
+        id: workspaceId,
+      },
+      relations: ['trustDomains'],
```

Comment:
I have a hard time understanding why we would add something to social logins but not to SSO login for example? Feels like there might be a wrong file name or architecture underneath?

Reply:
Yep the function is for all sso login including custom and social. I move the function.